pa/fin/tasks/update_tickers_statements.py

```diff
@@ -33,9 +34,24 @@ def update_tickers_statements(tickers_query):
         ticker.country = ticker_overview.get('Country', Ticker.DEFAULT_VALUE)
         ticker.industry = ticker_overview.get('Industry', Ticker.DEFAULT_VALUE)
         ticker.sector = ticker_overview.get('Sector', Ticker.DEFAULT_VALUE)
+
         pe_ratio = ticker_overview.get('PERatio')
         ticker.pe = None if pe_ratio == 'None' else pe_ratio
 
+        outstanding_shares = TickerStatement.objects \
+            .filter(name=Statements.outstanding_shares,
+                    ticker=ticker) \
+            .order_by('-fiscal_date_ending') \
+            .first()
+        if not outstanding_shares \
```

Comment:
Seems you do not use this variable. So probably it can be replaced with checking of the existence 

i.e. 
```python

 is_outstanding_shares = TickerStatement.objects \
            .filter(name=Statements.outstanding_shares,
                    ticker=ticker, 
                    <some additional filters here> ) \
            .order_by('-fiscal_date_ending') \
            .exists() # https://docs.djangoproject.com/en/3.1/ref/models/querysets/#django.db.models.query.QuerySet.exists 
```